app/store/env/go.js

```diff
@@ -2228,6 +2228,43 @@ YUI.add('juju-env-go', function(Y) {
       };
       // Call the original user callback.
       userCallback(transformedData);
+    },
+
+    /**
+      Makes a request of the websocket for the changeSet list of commands from
+      the supplied bundle YAML contents.
+
+      @method getChangeSet
+      @param {String} bundleYAML The bundle YAML file contents.
+      @param {Function} callback The user supplied callback to send the
+        changeset response to after post processing in handleGetChangeSet.
```

Comment:
_handleGetChangeSet
Anyway, this is more an internal detail. It would be nice to document exactly what data is passed to the callback.
In this case I'd suggest a list of changes (each one being a map) and a list of errors (each one being a human readable string).